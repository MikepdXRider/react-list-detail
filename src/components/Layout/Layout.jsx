import React from 'react'

export default function Layout({children}) {
    return (
        <>
            <header>
                {/* Maybe a little  */}
                <h1>Avatar List and Details</h1>
            </header>
            <main>
                {children}
            </main>
        </>
        
    )
}
