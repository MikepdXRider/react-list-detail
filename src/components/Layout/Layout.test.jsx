import { render } from "@testing-library/react";
import Layout from "./Layout.jsx";

it('renders layout component', () => {
    const {container} = render(
        <Layout />
    )

    expect(container).toMatchSnapshot();
})