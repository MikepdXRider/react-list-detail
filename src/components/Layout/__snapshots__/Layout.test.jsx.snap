// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders layout component 1`] = `
<div>
  <header>
    <h1>
      Avatar
    </h1>
    <p>
      The Last Airbender
    </p>
  </header>
  <main />
</div>
`;
